### PR TITLE
Added video for tracking package use and version management

### PR DESCRIPTION
**Pull request by ryan-blunden**
Video at https://vimeo.com/340774081

### CODE REVIEW

File: website/src/pages/product/code-search-navigation.tsx [hunk 1]
```diff
@@ -357,6 +357,8 @@ export default ((props: any) => (
                     Use Sourcegraph to find answers in code, faster!
                 </h2>
                 <Vimeo id={340491490} />
+                <p>&nbsp;</p>
```
Comment (christinaforney) [inline website/src/pages/product/code-search-navigation.tsx]: Why is this needed? Would it be better to add a margin or padding to a css class on the component? This is adding a second video to the black section? Do we need to introduce/title the two videos separately? Or we allow the splash page of the video to be that title (thinking about SEO...)?

**Comment by christinaforney** [comment]
Video looks good - I like the pauses you took to let the user read the message before performing the action.

File: website/src/pages/product/code-search-navigation.tsx (hunk 1 above)
Comment (ryan-blunden) [reply website/src/pages/product/code-search-navigation.tsx]: My latest commit fixes the hacky spacing code. Just in a rush.

This is not how I want to be long term, but for now, the videos are grouped under the **Use Sourcegraph to find answers in code, faster!** heading.

The video splash page is acting as a title for now. SEO is definitely a consideration but isn't a priority right now.,

Comment (christinaforney) [reply website/src/pages/product/code-search-navigation.tsx]: Sounds good!
